src/main/java/com/futurecollars/Task3/Task4/Age.java

```diff
@@ -0,0 +1,15 @@
+package com.futurecollars.Task3.Task4;
+
+public class Age {
+
+    public static boolean number(int Arg) {
+        if(Arg >= 18) {
```

Comment:
od kiedy to zmienne piszemy wielkimi literami?
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html